packages/mobile/src/identity/verification.ts

```diff
@@ -509,11 +509,15 @@ function* requestAttestations(
       numAttestationsRequestsNeeded
     )
 
+    // Using hard-coded gas value to avoid running gas estimation.
+    // Note: This is fragile and needs be updated if there are significant
+    // changes to the contract implementation.
     yield call(
       sendTransaction,
       approveTx.txo,
       account,
-      newTransactionContext(TAG, 'Approve attestations')
+      newTransactionContext(TAG, 'Approve attestations'),
+      150000
```

Comment:
Thanks for this! How would you feel about having these 4 numbers as constants so that it's easier to tell what they represent at a glance?

Reply:
Good idea! I changed to do this in 9c74575